client/packages/purchase_orders/src/DetailView/SidePanel/OtherSection.tsx

```diff
@@ -5,22 +5,25 @@ import {
   DetailPanelSection,
   PanelRow,
   PanelLabel,
-  BufferedTextArea,
+  TextArea,
 } from '@openmsupply-client/common';
 import { DonorSearchInput } from '@openmsupply-client/system/src';
 import { PurchaseOrderFragment } from '../../api';
-import { UpdatePurchaseOrderInput } from '../../api/hooks/usePurchaseOrder';
 
 // TODO: ShippingMethod have its own table. Need to migrate over before implementing this
 
 interface OtherSectionProps {
-  data?: PurchaseOrderFragment;
-  onUpdate: (input: Partial<UpdatePurchaseOrderInput>) => void;
+  draft?: PurchaseOrderFragment;
+  onDraftChange: (input: Partial<PurchaseOrderFragment>) => void;
+  onUpdate: (input: Partial<PurchaseOrderFragment>) => void;
+  onDebounceUpdate: (input: Partial<PurchaseOrderFragment>) => void;
```

Comment:
Do we have to pass these both everywhere? Can't we just update the draft, and let the hook handle the debounced saving? Feels like we're always calling both anyway?

Reply:
I was debating on doing that actually. But good to hear that Im not the only one thinking it. Will let the hook handle debouncing instead or could even just make it an option